content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -151,7 +151,7 @@ behavior.
 
 Fixed in the cert-manager PR [#4092][].
 
-[#4092]: https://github.com/jetstack/cert-manager/pull/4092 "Defaukt renewal changed from 30 days before to 2/3 of the duration before expiry"
+[#4092]: https://github.com/jetstack/cert-manager/pull/4092 "Default renewal changed from 30 days before expiry to 1/3 of the duration before expiry"
```

Comment:
TIL that these quoted strings are used as the `<a ...title="...">` content.
https://github.github.com/gfm/#link-reference-definition